src/map/map.page.component.tsx

```diff
@@ -76,6 +77,7 @@ export default class MapComponent extends React.Component<Props> {
             </StyledPaper>
           </Grid>
         )}
+        <Footer />
```

Comment:
```suggestion
        <Grid item xs={12}>
          <Footer />
        </Grid>
```
While loading, the footer will move to the right of the loading note otherwise.

Reply:
nice catch !